xlml/utils/subprocess_utils.py

```
@@ -82,3 +83,23 @@ def run_exec(
     logging.info("[subprocess] stdout: %s", res.stdout)
 
   return res.stdout
+
+
+def run_streaming(
+    cmd: str, log_command: bool = True, env: dict[str, str] | None = None
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)